elements/pfe-sass/extends/_extends.scss

```diff
@@ -47,3 +47,187 @@
     color: pfe-fetch(text) !important;
   }
 }
+
+// NOTES: 
+// - The interpoliation is for the benfefit of older systems
+// - These do not make use of the pfe-typography mixin because it 
+//   does not include the --pf-c-title--m CSS variables in the stack
+
+// TITLES
+%pfe-title--6xl {
```

Comment:
It would be great if the source of truth for these sets was in a map so that the conversion to design tokens will be easier when the time comes.  Something like:

```
$pfe-titles: (
    6xl: (
        font-family: pfe-var(font-family--heading),
        font-size: pfe-var(title--m-6xl--FontSize),
        line-height: pfe-var(line-height--heading),
        font-weight: pfe-var(font-weight--light),
        margin-bottom: pfe-var(content-spacer--heading--md)
    )
);

@each $size, $set in $pfe-title {
    %pfe-title--#{$size} {
        @each $key, $value in $set {
            @if $key != 'margin-bottom' {
                #{$key}: #{$value};
            }
            @else {
                &:not(:last-child) {
                    #{$key}: #{$value};
                }
            }
        }
    }
}
````